components/monitoring/base/configure-prometheus.yaml

```diff
@@ -118,9 +133,17 @@ spec:
     - --upstream=http://localhost:9090
     - --tls-cert=/etc/tls/private/tls.crt
     - --tls-key=/etc/tls/private/tls.key
+    - --request-logging=true
     - --cookie-secret=SECRET
     # see https://github.com/openshift/oauth-proxy/#delegate-authentication-and-authorization-to-openshift-for-infrastructure
     - --openshift-delegate-urls={"/":{"group":"","resource":"namespaces","verb":"get"}}
+    securityContext:
```

Comment:
we are already adding container security context. I believe we dont need pod security context again. or we can add pod securitycontext and remove the container security context as it is same for both the containers.

Reply:
apparently the security context defined in the Prometheus CR's `.spec` does not apply to the extra containers. When running `oc rollout restart statefulset/prometheus-prometheus`, I had multiple warning which were fixed after adding these security contexts.